resources/metadata/abapEnvironmentCreateTag.yaml

```diff
@@ -0,0 +1,160 @@
+metadata:
+  name: abapEnvironmentCreateTag
+  description: Creates a tag for a git repository to a SAP BTP ABAP Environment system
+  longDescription: |
+    Creates a tag for a git repository (Software Component) to a SAP BTP ABAP Environment system.
```

Comment:
To be more precise (as I undestand the implementation which I'm not sure)
* Creates Tags for all repositories defined in the respositories config file (mandatory parameter!) specifying VersionYAML with the version as its name (mandatory, cannot be disabled)
* In addition (not stand alone) for any further non related software component a tag can be created by providing repositoryName and CommitID and TagName and TagDescription [but keep in mind that a tag with this name/description will not only be created for this on related software component but for all software components specified via the repositories config file, too]
* if CreateTagForAddonProductVersion = true then for all repos specified in the repositories file name as well as the un related one a tag might be created if the config file contains valid entries for addonProduct and addonVersionYAML

Reply:
I made some changes to this behaviour:

- there is a flag for the component version and product version tags. if set to yes, all repositories specified in the repo.yml file will receive their respective tags
- all repositories (repo.yml and repositoryName + commitID) will be tagged with the "tagName"